### PR TITLE
chore: update default `Guild.filesize_limit` to 26214400 (25MB)

### PR DESCRIPTION
**Pull request by shiftinv**
## Summary

https://github.com/discord/discord-api-docs/commit/cfc4dcb835b2268274efe1ae5619ef7a769fb919

## Checklist

- [x] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `pdm lint`
    - [ ] I have type-checked the code by running `pdm pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: changelog/1005.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Update base :attr:`Guild.filesize_limit` to 25MB, matching recent API bump.
```
Comment (onerandomusername) [inline changelog/1005.misc.rst:1]:
```suggestion
Increase the default :attr:`Guild.filesize_limit` from 8MB to 25MB, matching the recent increase by Discord.
```

not sure about "default", maybe "unboosted" works better.

File: disnake/guild.py [hunk 2]
```diff
@@ -354,9 +354,9 @@ class Guild(Hashable):
...
 
     _PREMIUM_GUILD_LIMITS: ClassVar[Dict[Optional[int], _GuildLimit]] = {
-        None: _GuildLimit(emoji=50, stickers=5, bitrate=96e3, filesize=8388608),
-        0: _GuildLimit(emoji=50, stickers=5, bitrate=96e3, filesize=8388608),
-        1: _GuildLimit(emoji=100, stickers=15, bitrate=128e3, filesize=8388608),
+        None: _GuildLimit(emoji=50, stickers=5, bitrate=96e3, filesize=26214400),
+        0: _GuildLimit(emoji=50, stickers=5, bitrate=96e3, filesize=26214400),
+        1: _GuildLimit(emoji=100, stickers=15, bitrate=128e3, filesize=26214400),
         2: _GuildLimit(emoji=150, stickers=30, bitrate=256e3, filesize=52428800),
         3: _GuildLimit(emoji=250, stickers=60, bitrate=384e3, filesize=104857600),
```
Comment (onerandomusername) [inline disnake/guild.py:361]: is there an simpler way to write these numbers?

File: changelog/1005.misc.rst (hunk 1 above)
Comment (shiftinv) [reply changelog/1005.misc.rst:1]: yeah. It's also the limit for boost level 1 guilds, so "unboosted" doesn't quite work, but "default" seems fine.

File: disnake/guild.py (hunk 2 above)
Comment (shiftinv) [reply disnake/guild.py:361]: Those numbers correspond to `25 * 1024 * 1024`, `50 * ...` and `100 * ...` respectively, but I don't think it's necessary to rewrite them.

**Review by onerandomusername — APPROVED** [review]

**Pull request merged**